src/api/rest/v1/ds/routes.ts

```diff
@@ -7,6 +7,8 @@ router.get(/get\/(.*)\/(.*)$/, controller.getById)
 router.post(/query\/(.*)$/, controller.query)
 router.delete(/([^\/]*)$/, controller.delete)
 router.get(/watch\/(.*)$/, controller.watch)
+router.post(/save\/(.*)$/, controller.create)
+router.put(/save\/(.*)\/(.*)$/, controller.update)
```

Comment:
Curiosity: why the regexp rather than the `/:schema/:recordId` notation?

Also, for the RESTFull, could we ditch the `/save` part of the route?

Reply:
Yep, good idea